### PR TITLE
Added logging to verify startDateTime and endDateTime and their value…

### PR DESCRIPTION
**Pull request by ange47rm**
Added logging to verify startDateTime and endDateTime and their values prior to going to DRS

### CODE REVIEW

File: HousingRepairsSchedulingApi/Startup.cs [hunk 1]
```diff
@@ -48,19 +49,9 @@ public void ConfigureServices(IServiceCollection services)
...
-                {
-                    var drsOptions = sp.GetRequiredService<IOptions<DrsOptions>>();
-                    var appointmentSearchTimeSpanInDays = drsOptions.Value.SearchTimeSpanInDays;
-                    var appointmentLeadTimeInDays = drsOptions.Value.AppointmentLeadTimeInDays;
-                    var maximumNumberOfRequests = drsOptions.Value.MaximumNumberOfRequests;
-                    return new DrsAppointmentGateway(sp.GetService<IDrsService>(),
-                        5, appointmentSearchTimeSpanInDays, appointmentLeadTimeInDays, maximumNumberOfRequests);
-                }
-            );
+            services.AddTransient<IDrsService, DrsService>();
```
Comment (Asura5) [inline HousingRepairsSchedulingApi/Startup.cs:52]: This isn't logging - why has this changed?

File: HousingRepairsSchedulingApi/Controllers/AppointmentsController.cs [hunk 2]
```diff
@@ -5,7 +5,8 @@ namespace HousingRepairsSchedulingApi.Controllers
     using System.Threading.Tasks;
     using Amazon.Lambda.Core;
     using HousingRepairsSchedulingApi.UseCases.Interfaces;
-    using Microsoft.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Mvc;
```
Comment (Asura5) [inline HousingRepairsSchedulingApi/Controllers/AppointmentsController.cs:8]: Don't think this is needed

File: HousingRepairsSchedulingApi/Startup.cs (hunk 1 above)
Comment (ange47rm) [reply HousingRepairsSchedulingApi/Startup.cs:52]: In line 82 when DrsAppointmentGateway is instantiated the instance of logger (line 72) is passed in. Do you think it'd be better not to have a separate InitializeAppointmentsGateway method?

Comment (Asura5) [reply HousingRepairsSchedulingApi/Startup.cs:52]: But this is changing the startup - there's an extra call to var provider = services.BuildServiceProvider();

Comment (ange47rm) [reply HousingRepairsSchedulingApi/Startup.cs:52]: I think I see what you mean, I'll try and refactor

Comment (ange47rm) [reply HousingRepairsSchedulingApi/Startup.cs:52]: @Asura5 I've made a change now and pushed, checks are green, please see the below:
https://github.com/LBHackney-IT/HousingRepairsSchedulingApi/pull/17/commits/69017aff0086d5c3483cc7a074bb963e826a5e03

**Review by Asura5 — APPROVED** [review]

**Pull request merged**